x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/dashboards/enable_risk_score_redirect.cy.ts

```diff
@@ -36,18 +32,10 @@ describe(
       visit(ENTITY_ANALYTICS_URL);
     });
 
-    it('host risk enable button  should redirect to entity management page', () => {
-      cy.get(ENABLE_HOST_RISK_SCORE_BUTTON).should('exist');
+    it('risk enable button should redirect to entity management page', () => {
```

Comment:
Do we still need the above feature flag?

Reply:
[remove unused feature flag from test](https://github.com/elastic/kibana/pull/201810/commits/818cbdd3f4b70af23f9beb7f634a7b3fbd84e87a)